tests/travis-ci/fixtures/phpunit.xml.dist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="core/tests/bootstrap.php"
+         colors="true"
+         beStrictAboutTestsThatDoNotTestAnything="true"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutChangesToGlobalState="true"
+         checkForUnintentionallyCoveredCode="false"
+         printerClass="\Drupal\Tests\Listeners\HtmlOutputPrinter"
+>
+    <php>
+        <!-- Set error reporting to E_ALL. -->
+        <ini name="error_reporting" value="32767"/>
+        <!-- Do not limit the amount of memory tests take to run. -->
+        <ini name="memory_limit" value="-1"/>
+        <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
+        <env name="SIMPLETEST_BASE_URL" value="http://localhost:8888"/>
+        <!-- Example SIMPLETEST_DB value: mysql://username:password@localhost/databasename#table_prefix -->
+        <env name="SIMPLETEST_DB" value="sqlite://localhost//tmp/test.sqlite"/>
+        <!-- Example SIMPLETEST_SPARQL_DB value: sparql://127.0.0.1:8890/ -->
+        <env name="SIMPLETEST_SPARQL_DB" value="sparql://127.0.0.1:8890/sparql"/>
```

Comment:
I wonder how this works w/o `?module=sparql_entity_storage` query strung. See https://www.drupal.org/node/3123251